### ISSUE TITLE
Allow piping

### ISSUE THREAD

**Issue by OliverJAsh** [kimmobrunfeldt/concurrently#41]
When I try to use the `|` pipe, I get weird errors. It seems concurrently considers the pipe to be an argument of the command.

```
$ concurrently --raw 'tail -f temp | grep foo'
tail -f temp | grep foo [ 'tail', '-f', 'temp', '|', 'grep', 'foo' ]
tail: |: No such file or directory
tail: grep: No such file or directory
tail: foo: No such file or directory
```

If I run the command without concurrently it works fine.

**Comment by OliverJAsh** [comment]
Let me know how you think this needs to be fixed, happy to contribute :-)

**Comment by steve-todorov** [comment]
The same thing happens with ampersand `&`

`
$ concurrently "sleep 5s && npm run protractor"
 sleep: invalid time interval '&&'
 sleep: invalid time interval 'npm'
 
 sleep: invalid time interval 'run'
 sleep: invalid time interval 'protractor'
 Try 'sleep --help' for more information.
 sleep 5s && npm run protractor exited with code 1
`

ping @kimmobrunfeldt

**Comment by kimmobrunfeldt** [comment]
Hi, 

This is happening because the underlying implementation how these are fed to the system shell is not very good. If you happen to know a good cross-platform library which would spawn "the best" available shell with support for piping etc, I'd be glad to start using it in concurrently.

**Comment by kimmobrunfeldt** [comment]
Could you test if this has been fixed by installing 3.0.0-dev version: `npm i -g concurrently@3.0.0-dev` ? 
It should be better now because it is using https://github.com/kimmobrunfeldt/spawn-default-shell for spawning shell.

**Comment by kimmobrunfeldt** [comment]
These issues should be now fixed in 3.0.0-rc1. You can test by installing it: `npm i -g concurrently@3.0.0-rc1`. Please open a new issue to concurrently repo if this is not the case. I'll release the 3.0.0 soonish.

**Comment by steve-todorov** [comment]
Hi, 

Sorry for the late response, but I was busy at work. I've tested the rc version and it seems to be working fine. Thanks for the great work and fast response!
